lighthouse-cli/test/smokehouse/lighthouse-runners/cli.js

```diff
@@ -54,7 +54,7 @@ async function internalRun(url, tmpPath, configJson, isDebug) {
   const artifactsDirectory = `${tmpPath}/artifacts/`;
 
   const args = [
-    'lighthouse-cli/index.js',
+    `${__dirname}/../../../index.js`, // 'lighthouse-cli/index.js'
```

Comment:
needs to be absolute so it works from inside `node_modules/lighthouse/`, otherwise it tries to run it from the root of the parent project (this is why publisher ads had to [change the cwd](https://github.com/googleads/publisher-ads-lighthouse-plugin/blob/2df6b33bcecda4ef5f4948be3d906511c29ca0ae/lighthouse-plugin-publisher-ads/test/smoke/run-smoke.js#L49) for their tests)